Bottom: 0822b1247832f6d88d7d6f78879c93bd511bf4cf
Top:    d0822be85c5a453583a009ff631a9c5095ce33f3
Author: Markus Hitter <mah@jump-ing.de>
Date:   2011-01-24 11:15:02 +0100

configure.ac: remove the now obsolete test for gethostname().

No longer required since commit 00c556b02dfa0113eab62b894eff4cf5d55b8e5f

Reviewed-by: Peter Clifton <pcjc2@cam.ac.uk>
Affects-bug: lp-703914


---

diff --git a/configure.ac b/configure.ac
index e405819..8fde977 100644
--- a/configure.ac
+++ b/configure.ac
@@ -650,7 +650,7 @@ AC_CHECK_FUNCS(strerror)
 AC_CHECK_FUNCS(regcomp re_comp)
 AC_CHECK_FUNCS(logf expf rint)
 AC_CHECK_FUNCS(vsnprintf)
-AC_CHECK_FUNCS(getpwuid gethostname getcwd)
+AC_CHECK_FUNCS(getpwuid getcwd)
 AC_CHECK_FUNCS(random)
 AC_CHECK_FUNCS(stat)
